packages/docs/src/routes/api/qwik/index.md

```diff
@@ -1214,6 +1214,24 @@ export interface QwikFocusEvent<T = Element> extends SyntheticEvent<T, NativeFoc
 
 ## QwikIntrinsicElements
 
+The interface holds available attributes of both native DOM elements and custom Qwik elements. An example showing how to define a customizable wrapper component:
+
+```tsx
+import { component$, Slot, type QwikIntrinsicElements } from "@builder.io/qwik";
+
+type WrapperProps = {
+  attributes?: QwikIntrinsicElements["div"];
+};
+
+const Wrapper = component$<WrapperProps>(({ attributes }) => {
```

Comment:
just a thought - maybe we could use an  `export default component$(...)` here instead to make it a working example?

Reply:
Sure, I can make that change soon.